### PR TITLE
Modify the capabilities constants to match header files like other constants

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: config-linux.md [hunk 1]
```diff
@@ -7,9 +7,8 @@ in order to configure these various kernel features.
 
 ## Capabilities
 
-Capabilities is an array that specifies Linux capabilities that can be provided to the process
-inside the container. Valid values are the string after `CAP_` for capabilities defined
-in [the man page](http://man7.org/linux/man-pages/man7/capabilities.7.html)
+Capabilities is an array that specifies Linux capabilities that can be provided to the process inside the container.
+Valid values are the strings for capabilities defined in [the man page](http://man7.org/linux/man-pages/man7/capabilities.7.html)
```
Comment (LK4D4) [inline config-linux.md]: Maybe add (e.g. CAP_SYS_ADMIN), because those aren't strings in man.

Comment (mrunalp) [inline config-linux.md]: I modified the example json.

**Comment by vbatts** [comment]
LGTM

**Comment by wking** [comment]
Why require the leading CAP_?  Isn't that already clear from the
context (linux.capabilities)?

**Comment by philips** [comment]
to-may-to to-mah-to, is there a strong argument one way or the other?

**Comment by mrunalp** [comment]
@philips Thinking it might be better to explicit in these cases and just match the values defined in the header files at the expense of typing few more characters.

Rebased.

**Comment by vishh** [comment]
LGTM

**Comment by philips** [comment]
lgtm

**Pull request merged**